app.js

```diff
@@ -10,29 +10,40 @@ const tryAgain = document.getElementById('try-again');
 let randomNumber = Math.ceil(Math.random() * 20);
 let guessesLeft = 4;
 //console.log(userNumber.value, 'value of my input');
+guessNow.classList.remove('hidden');
 
 console.log(randomNumber);
 // set event listeners to update state and DOM
 guessNow.addEventListener('click', () => {
-    if(guessesLeft <= 0){
+    if (guessesLeft <= 0){
         alert('You ran out of guesses!');
         return;
     }
     //reduce guess count
     guessesLeft--;
 
-    console.log(guessesLeft);
-    //console.log(userNumber.value, 'value of input');
+    const guessResults = isNumber(userNumber.valueAsNumber, randomNumber);
```

Comment:
you could set const number = userNumber.valueAsNumber and then use `number` wherever you need it to minimize some code